bin/reth/src/node/mod.rs

```diff
@@ -119,8 +123,16 @@ impl Command {
         let consensus = Arc::new(BeaconConsensus::new(self.chain.consensus.clone()));
         let genesis_hash = init_genesis(db.clone(), self.chain.genesis.clone())?;
 
+        let envs = &self.bootnodes;
+
         let network = config
-            .network_config(db.clone(), chain_id, genesis_hash, self.network.disable_discovery)
+            .network_config(
+                db.clone(),
+                chain_id,
+                genesis_hash,
+                self.network.disable_discovery,
+                &envs,
```

Comment:
```suggestion
                self.bootnodes,
```

Reply:
Alright, I’ve a better understanding now, I’ll make the changes. 
This breaks the networkconfig builder in other places, cause of the extra parameter, can I fix those up too